### PR TITLE
Coding Challenge 181: Weighted Voronoi Stippling

### PR DESCRIPTION
**Pull request by shiffman**
Starting a branch, any help with example images, example links, timecodes, description, references, video links, etc. is welcome!

CC-181-Image-Stippling-Thumbnail-Black-Dots-Black-BG

### CODE REVIEW

File: content/videos/challenges/181-image-stippling/index.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
+{
+  "title": "Weighted Voronoi Stippling",
+  "videoNumber": "181",
+  "description": "TBD",
+  "videoId": "TBD",
+  "nebulaSlug": "codingtrain-coding-challenge-181-image-stippling",
+  "date": "2024-02-22",
+  "languages": ["p5.js", "JavaScript"],
+  "topics": ["daulanay triangulation", "image stippling", "voronoi", "sweep algorithm", "Lloyd's algorithm"],
```
Comment (fturmel) [inline content/videos/challenges/181-image-stippling/index.json:9]: Little typo, should be "Delaunay triangulation"

File: content/videos/challenges/181-image-stippling/index.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  "canContribute": true,
+  "relatedChallenges": ["c4-worley-noise", "33-poisson-disc-sampling"],
+  "timestamps": [
+    { "time": "0:00", "title": "Hello!" },
+    { "time": "0:37", "title": "Delaunay triangulation" },
+    { "time": "2:31", "title": "d3-daulanay" },
```
Comment (fturmel) [inline content/videos/challenges/181-image-stippling/index.json:15]: d3-delaunay

File: content/videos/challenges/181-image-stippling/index.json (hunk 1 above)
Comment (kfahn22) [reply content/videos/challenges/181-image-stippling/index.json:9]: Thanks!

**Comment by shiffman** [comment]
Allright, I think I've got everything in here! Video will go live tomorrow. Any additional corrections, additions, suggestions, etc can be filed in separate issues or pull requests!

**Pull request merged**